components/sections/Hero/page.tsx

```diff
@@ -9,15 +11,9 @@ const HeroSection = () => {
           Community to bring together and empower fellow Muslims in tech
         </p>
         <div className="flex flex-col sm:flex-row gap-4">
-          <button className="bg-blue-600 text-white font-bold py-3 px-6 rounded-lg hover:bg-blue-700 transition-colors">
-            Get Started
-          </button>
-          <button className="bg-transparent text-blue-600 font-semibold py-3 px-6 rounded-lg border border-blue-600 hover:bg-blue-600 hover:text-white transition-colors">
-            Learn More
-          </button>
-          <button className="bg-transparent text-blue-600 font-semibold py-3 px-6 rounded-lg border border-blue-600 hover:bg-blue-600 hover:text-white transition-colors">
-            Discord
-          </button>
+          <GetStarted></GetStarted>
```

Comment:
Why do we need get started button? What is the purpose?

Reply:
it can prolly lead users to about me page or something similar that will allow the user to know what is MIT for. I feel like most organizations has Get Started button

Reply:
ok